README.md

```diff
@@ -27,326 +27,36 @@ The OBS source code repository is hosted on [Github](http://github.com/opensuse/
 ## Setup
 There are 3 scenarios for which you can setup an OBS instance. Running it in *production* for your users, for *development* on it and for executing the *test* suite.
```

Comment:
Actually, I would also drop development here, just pointing developers to the CONTRIBUTING file.
Just saying "production" is what you want.